profile.yaml

```diff
@@ -0,0 +1,249 @@
+openapi: 3.0.0
+info:
+  title: Simple REST APIs
+  description: This is profile API.
+  termsOfService: http://swagger.io/terms/
```

Comment:
Let's not use entries for which we don't have information at this point. For example, termsOfService, license, version, etc.